cg/services/store/rna_dna_collections_service/rna_dna_collections_service.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from cg.constants import DNA_PREP_CATEGORIES, DNA_WORKFLOWS_WITH_SCOUT_UPLOAD
+from cg.exc import CgDataError
+from cg.store.models import Case, Customer, Sample
+from cg.store.store import Store
+
+LOG = logging.getLogger(__name__)
+
+
+class RNADNACollectionsService:
```

Comment:
This service seems to be a filter for dna rna cases. Why can we not solve this by adding direct database queries in the store?